hub/config/src/main/java/uk/gov/ida/hub/config/domain/IdentityProviderConfig.java

```diff
@@ -127,7 +131,16 @@ public Boolean isEnabledForSingleIdp() {
     }
 
     public boolean isOnboardingForTransactionEntity(String transactionEntity) {
-        return this.getOnboardingTransactionEntityIds().contains(transactionEntity);
+        boolean isOnboarding = !getOnboardingTransactionEntityIds().isEmpty();
```

Comment:
I don't feel like this line makes what's happening more clear :-)

Reply:
Really, this is just pulling around from one place to another. I agree it's not very clear, but for this stage I didn't what to change what the code does. The existing method wasn't being used, and was squatting the name that made it consistent with the other predicates. I plan on making another update that makes the code clearer more straightforward to read.

Reply:
Sure, but it'd still do the same thing with the existing implementation, I don't think you need the boolean at all?

Reply:
No, this does change the behaviour of this method. The new behaviour returns true if the list is empty.

As I say, I do think the intended behaviour could be much clearer, but I'm being cautious and not making any changes to the boolean logic being executed from the old predicates classes until I've fully understood it. Getting rid of the `Predicate` class cruft should help make it clearer what the actual intent is.

Reply:
Okay, in which case the function name now needs to change, because even now having read it 3 or 4 times I was confused :-)

Reply:
Removal of deprecated endpoints also removes this method.